### PR TITLE
Add FASTQE as a Galaxy tool 🎉🔬🦠🧫🧪💉🧬🥳

### PR DESCRIPTION
**Pull request by hexylena**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Saw @lonsbio's BCC abstract and thought Galaxy needed it.

I reckon it'll be good for training materials, much more friendly than ASCII ;)

picture of fastqe's output formatted for Galaxy as an HTML dataset

### CODE REVIEW

File: tools/fastqe/fastqe.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<tool id="fastqe" name="FASTQE" version="0.1.1">
+	<description></description>
```
Comment (nsoranzo) [inline tools/fastqe/fastqe.xml:2]: Replace tabs with spaces everywhere please?

**Comment by hexylena** [comment]
Is that a standard that I missed that I should conform to? My personal
preference is for tabs.

man. 13. jul. 2020, 19:07 skrev Nicola Soranzo <notifications@github.com>:

File: tools/fastqe/fastqe.xml (hunk 1 above)
Comment (hexylena) [reply tools/fastqe/fastqe.xml:2]: Edit: Yes, it is a standard. Ok, I can do that.

File: tools/fastqe/fastqe.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <command detect_errors="exit_code">
+    <![CDATA[
+
+    #for i in $input:
+        ln -s '$i' '$i.element_identifier' &&
+    #end for
+
+    echo "<html><head><title>FASTQE Report 🤔</title></head><body><h1>FASTQE Report 🤔</h1>" > '$output' &&
+
+    /home/hxr/arbeit/miniconda/envs/test/bin/fastqe
```
Comment (bgruening) [inline tools/fastqe/fastqe.xml:15]: This looks not good :)

Comment (hexylena) [reply tools/fastqe/fastqe.xml]: 😨

File: tools/fastqe/fastqe.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<tool id="fastqe" name="FASTQE" version="0.1.1">
+    <description></description>
```
Comment (bgruening) [inline tools/fastqe/fastqe.xml:2]: visualize fastq files with emoji's  ?

File: tools/fastqe/fastqe.xml (hunk 2 above)
Comment (nsoranzo) [reply tools/fastqe/fastqe.xml]: Love the German there :D

**Comment by lonsbio** [comment]
Yesssssssss! Great work  @hexylena 

I had tried something rudimentary  for Galaxy but this is just amazing. Much better than my novice attempts.  The HTML wrapping looks great. 

A new version of FASTQE is due next week, is it just a matter of doing a  PR to this file with the new version number and features once the new version is in pip/conda?

**Comment by hexylena** [comment]
Heya @lonsbio, bril! glad you like it.

Yep, once the conda package is updated, then we'll just have to bump the version here (+ the requirements block) and we're sorted.

Looking forward to the new release!

**Pull request merged**